city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,134 @@
+            items = year_list.xpath('.//following-sibling::ul[1]/li').extract()
+            for item in items:
+                #print(item)
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    description='',
+                    classification=self._parse_classification(item),
+                    start=self._parse_start(item,year),
+                    end=None,
+                    #all_day=self._parse_all_day(item),
+                    time_notes="See agenda for meeting time",
+                    location=self._parse_location(item),
+                    #links=self._parse_links(item),
+                    #source=self._parse_source(response),
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+                yield meeting'''
+
+    def _parse_title(self, item):
+
+        """Parse or generate meeting title."""
+        pattern= re.compile('-(.*(Committee|Meeting))',re.IGNORECASE)
+        title_list = re.findall(pattern, item)
+        if title_list == None:
+            return 'Board of Trustees'
+        title, ign=title_list[-1]
+        return title.replace("<strong>","").replace("</strong>","").replace("\xa0"," ")
+
+    def _parse_classification(self, title):
+        """Parse or generate classification from allowed options."""
+        if 'committee' in title.lower():
+            return COMMITTEE
+        if "meeting" in title.lower():
+            return BOARD
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item,year):
+        """Parse start datetime as a naive datetime object."""
+        pattern = re.compile('([a-z]+)(,|\u00a0|\u0020)+([a-z]+)(,|\u00a0|\u0020)+([0-9]{1,2})',re.IGNORECASE)
+        day, ign1, month, ign2, date = re.findall(pattern,item)[0]
+        date_obj = datetime.datetime(int(year),month_mapping[month],int(date),13,00)
+        return date_obj
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        if "**" in item:
+            return {
+                "address": "",
+                "name": "Jacob H. Carruthers Center",
+        }
+        if "*" in item:
+            return {
+                "address": "",
+                'name'   : "El Centro"
+            }
+
+        return {
+            "address": "5500 North St. Louis Avenue, Chicago, Ill., 60625",
+            "name"   : 'Northeastern Illinois University'
+        }
+
+    def _parse_links(self):
+        """Parse or generate links."""
+
+        link_response = scrapy.Request("https://www.neiu.edu/about/board-of-trustees/board-meeting-materials")
```

Comment:
Since you need the materials to associate them to meetings, you should have this URL in `start_urls`, and then once it's parsed `yield` a request to the calendar page. You can see what this looks like in [`cook_housing`](https://github.com/City-Bureau/city-scrapers/blob/75a4d974e4e35fd3cc349c75d1cd68f451b96b9c/city_scrapers/spiders/cook_housing.py#L22-L43)